rhobs/README.md

```diff
@@ -0,0 +1,18 @@
+# RHTAP Alerting Rules
+RHTAP alerting rules are deployed to RHOBS (Observatorium) using app-interface.
+The rules are defined inside the [alerting](alerting) directory and are referenced using
```

Comment:
```suggestion
The rules are defined inside the [alerting](alerting) directory and are referenced using
```
this link is not there.

Reply:
I lost track of this PR. I think I accidentally marked the ticket as done.
Yes. It was written before the directory was moved, and I think it was eventually moved elsewhere. I'll look into that.

Reply:
ok. I see now that I replaced it with another PR instead: https://github.com/redhat-appstudio/o11y/pull/54
I'll close this one.
Thank you for reminding me :)